spec/shortcode/presenter_spec.rb

```diff
@@ -101,6 +102,13 @@
       end
     end
 
+    context "when initialize exists on non-Object ancestor class" do
```

Comment:
Would be good to have an unhappy path test as well that checks for an exception when neither a parent or child class define the `initialize` method.

Reply:
Good call. Will add one.